lib/discovery_api_web/controllers/dataset_preview_controller.ex

```diff
@@ -1,14 +1,14 @@
 defmodule DiscoveryApiWeb.DatasetPreviewController do
   use DiscoveryApiWeb, :controller
-  alias DiscoveryApiWeb.DatasetPrestoQueryService
+  alias DiscoveryApiWeb.Services.PrestoService
```

Comment:
Same as above comment.
```suggestion
  alias DiscoveryApiWeb.Services.Presto
```